### DISCUSSION TITLE
Is there any way to exclude the translation strings sent to transifex?

### DISCUSSION THREAD

**Discussion by chenrui333** [transifex/cli#156] [General]
Right now, we are doing the v2 to v3 upgrade using the new API and v3 endpoint.

With v2 API, we had some custom in house process of pushing the translation strings into transifex, that code has some functionality of being able exclude some translation string patterns. While I want to get rid of some in house custom code in favor of the new cli, but I am not sure whether this is supported in the new CLI. Thanks!

**Comment by codegaze** [comment]
Hello @chenrui333,

CLI works on a file level with some specific filters for translation strings (like get only translated, reviewed etc) and that's about it. 

I don't see making changes to that logic in the near future, but feel free to share what you'd like to achieve in case we can share some input. As always, if you are comfortable with Go you can take parts of the functionality of the CLI and make it your own. There is an internal "package" of transifex APIv3 in this repo.

**Comment by chenrui333** [comment]
> I don't see making changes to that logic in the near future, but feel free to share what you'd like to achieve in case we can share some input.

understand.

On top of that, what is your recommendation on how to exclude the strings for translations (would that be that just on the file level, like dumping the strings that dont need to be translated into some sort of special file)
